src/Glpi/Form/Destination/AbstractCommonITILFormDestination.php

```diff
@@ -83,6 +84,16 @@ final public function createDestinationItems(
             'entities_id' => $form->fields['entities_id']
         ];
 
+        // Template field must be computed before applying predefined fields
+        $target_itemtype = static::getTargetItemtype();
+        $template_class = (new $target_itemtype())->getTemplateClass();
+        $template_field = new TemplateField($template_class);
+        $input = $template_field->applyConfiguratedValueToInputUsingAnswers(
```

Comment:
Maybe you could remove the template field from the list after this (to avoid computing the field twice) ?